build/r_gcs_snippets_menu_config.yml

```diff
@@ -1,9 +1,9 @@
-Google Bucket snippets:
+All of Us R and Cloud Storage snippets:
   - Documentation: https://github.com/all-of-us/workbench-snippets
   - divider: divider
   - (1) Setup: gcs-snippets/snippets_setup.R
-  - (2) Copy file to or from Google Bucket:
-    - copy_data_to_google_bucket.R: gcs-snippets/copy_data_to_google_bucket.R
-    - copy_file_from_google_bucket.R: gcs-snippets/copy_file_from_google_bucket.R
+  - (2) Copy file to or from Workspace Bucket:
+    - copy_data_to_workspace_bucket.R: gcs-snippets/copy_data_to_workspace_bucket.R
```

Comment:
Existing issue, but is the difference between "data" and "file" naming here intentional?

Reply:
Yes, because users will start with a dataframe and save that in Google Bucket. Whereas they start with a file from a bucket and load it in a dataframe